views/create-event.html

```diff
@@ -16,7 +16,116 @@
 	<form id="create-event" method="POST">
     <label for="title">Event title:</label>
     <input type="text" name="title">
+    
+    <label for="location">Event location:</label>
+    <input type="text" name="location">
+    
+    <label for="image">Event image:</label>
+    <input type="text" name="image">
+    
+    <label for="year">Event year:</label>
+    <input type="text" name="year">
+    
+    <label for="month">Event month:</label>
+    <input type="text" name="month">
+    
+    <label for="day">Event day:</label>
+    <input type="text" name="day">
+    
+    <label for="hour">Event hour:</label>
+    <input type="text" name="hour">
+    
+    <label for="minute">Event minute:</label>
+    <input type="text" name="minute">
+    
+    <input id="submit" type="submit">
   </form>
+  
+  <select name="month" form="create-event">
+   <option value="0" selected>2015</option> 
+   <option value="1" >2016</option>
+  </select>
+  
+  <select name="month" form="create-event">
+   <option value="0" selected>January</option> 
+   <option value="1" >February</option>
+   <option value="2" >March</option>
+   <option value="3" >April</option>
+   <option value="4" >May</option>
+   <option value="5" >June</option>
+   <option value="6" >July</option>
+   <option value="7" >August</option>
+   <option value="8" >September</option>
+   <option value="9" >October</option>
+   <option value="10" >November</option>
+   <option value="11" >December</option>
+  </select>
+  
+  <select name="day" form="create-event">
+    <option value="1" selected>01</option> 
```

Comment:
A better way to do this would be to construct an array of these values in javascript, and then use the `for` template helper `http://mozilla.github.io/nunjucks/templating.html#for` so that this does not need to be repeated multiple times for each month, day, etc in the template.